packages/fx-core/src/plugins/resource/appstudio/v3/plugin.ts

```diff
@@ -84,11 +84,11 @@ export class AppStudioPluginImpl {
         appStudioToken!,
         ctx.logProvider
       );
-      ctx.logProvider?.info(`Teams app created: ${appDefinition.appId}`);
+      ctx.userInteraction?.showMessage("info", `Teams app created: ${appDefinition.appId}`, false);
```

Comment:
For VSC, userInteraction will show the notification dialog which means the dialog(app created/updated) will be displayed when user provision, publish, localdebug, .... Just want to make sure whether that's expected behavior.

Reply:
Just checked with @nliu-ms that it's ok.